src/main/java/lotto/constant/Rank.java

```diff
@@ -0,0 +1,47 @@
+package lotto.constant;
+
+public enum Rank {
+    FIFTH(5, 2, 0, "꽝"),
+    FOURTH(4, 3, 5000, "3개 일치 (5,000원) - "),
+    THIRD(3, 4, 50000, "4개 일치 (50,000원) - "),
+    SECOND(2, 5, 1500000, "5개 일치 (1,500,000원) - "),
+    FIRST(1, 6, 2000000000, "6개 일치 (2,000,000,000원) - ");
```

Comment:
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요!
```suggestion
    FIRST(1, 6, 2_000_000_000, "6개 일치 (2,000,000,000원) - ");
```

Reply:
_를 넣으면 String이 되는거 아닌가 했는데, type은 변하지 않네요 😄 

좋은 의견 감사합니다.